client/me/security-2fa-disable/index.jsx

```diff
@@ -156,7 +156,7 @@ class Security2faDisable extends Component {
 						{ translate(
 							'To verify that you wish to disable two-step authentication, ' +
 								'enter the verification code from your device or a backup code, ' +
-								'and click "Disable two-step."'
+								'and click "Disable Two-Step Authentication."'
```

Comment:
If you're feeling really fancy, we like to use nice quote marks.

```suggestion
								'and click “Disable two-step authentication.”'
```